### ISSUE TITLE
Setting global volume before loading not taking effect

### ISSUE THREAD

**Issue by celynwalters** [goldfire/howler.js#1229]
I'm loving this library, thanks!
This is not #233.

I'm setting the global volume before loading any sounds, using:
```js
Howler.volume(slider.value / 100.0);
```
When querying, it reports as expected (I have the slider value set to 10/100).
This is true at every point (now, after loading a sound, when the sound is playing).
```js
console.log(Howler.volume());
=> 0.1
```
But it's actually playing at full volume.
I know this because once something is playing I can move the slider 1px which fires off `Howler.volume` again, and it immediately goes quieter back to the requested value.

It seems to be fine if I unload, then load a new track and play it, so I think it's only before Howler loads a track for the first time.

**Comment by defmech** [comment]
I have exactly the same issue at the moment. Setting `Howler.volume(0);` from a config file > Loading sounds > Play sound which plays at fill volume.

**Comment by goldfire** [comment]
This should be fixed with #1210.
